### ISSUE TITLE
sms problem

### ISSUE THREAD

**Issue by pepe-god** [nfdz/Cryptool#57]
I tried to use it over sms with my friend, but it sends sms but cannot read the incoming sms and automatically decode them.

**Comment by nfdz** [comment]
@pepe-god Thank you for reporting this issue. May I ask you for more details?
- The phone sends the SMS?
- The phone receives the SMS?
- Is it a long SMS? (more than 70 characters)

**Comment by pepe-god** [comment]
@nfdz Yes, the phone sends sms but does not receive it back, I have to solve it manually. The message is longer than 70 characters. My friend and I have the same problem on our Redmi note 10s (rosemary) and we use the same phone. We haven't had the opportunity to use it much, very good application if the problem is solved.

**Comment by pepe-god** [comment]
In the latest version of the application, we sent a text message in the same way, but when the sms comes from the other side, the application does nothing, there is sending a message, there is no reading the message, I also gave the necessary permissions, but in vain. When I examine the message manually, there is no problem there.
